nextcloud/docker-compose.yaml

```diff
@@ -48,29 +47,17 @@ services:
       - no-new-privileges:true
     volumes:
       - data:/var/www/html
+      #- photos:/var/www/html/data
     environment:
       - POSTGRES_DB=nextcloud
       - POSTGRES_USER=nextcloud
       - POSTGRES_HOST=postgres
-      - POSTGRES_PASSWORD=${DATABASE_PASSWORD}
+      - POSTGRES_PASSWORD=${NEXTCLOUD_DATABASE_PASSWORD}
       - REDIS_HOST=redis
-      - OBJECTSTORE_S3_BUCKET
-      - OBJECTSTORE_S3_SSL=true
-      - OBJECTSTORE_S3_USEPATH_STYLE=true
-      - OBJECTSTORE_S3_AUTOCREATE=true
-      - OBJECTSTORE_S3_KEY
-      - OBJECTSTORE_S3_SECRET
-      - OBJECTSTORE_S3_HOST
-      - OBJECTSTORE_S3_PORT=443
```

Comment:
Don't these need to be set from the NEXTCLOUD_OBJECTSTORE_S3_* variables?

Reply:
@EnigmaCurry, the OBJECTSTORE_S3_* env vars are set in docker-compose.instance.yaml if NEXTCLOUD_PRIMARY_STORAGE=S3 Bucket because if they're in docker-compose.yaml but the user doesn't enter values (i.e., doesn't want to use S3 for primary storage), Nextcloud sees that the env vars exist and tries to install primary storage to S3, and will fail because there are no values.

https://github.com/EnigmaCurry/d.rymcg.tech/pull/247/files#r1660169968

Reply:
see comment below - resolve if satisfied.